sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-kafka/README.md

```diff
@@ -64,9 +64,16 @@ Running this sample will be charged by Azure. You can check the usage and bill a
 
 1.  Delete the resources on [Azure Portal][azure-portal] to avoid unexpected charges.
 
+## Troubleshooting
 
```

Comment:
beside this troubleshooting doc, could you also check if this may help https://www.baeldung.com/spring-boot-failure-analyzer

Reply:
Ok,

Reply:
I just used the `FailureAnalyzer` feature, I think it might not be appropriate, because there are too many `UnsupportedVersionException` [exceptions](https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/errors/UnsupportedVersionException.java), and we have no way to pinpoint the exception that is currently concerned, which is different from this kind of `PortInUseException`.